cmd2/parsing.py

```diff
@@ -0,0 +1,387 @@
+#
+# -*- coding: utf-8 -*-
+"""Statement parsing classes for cmd2"""
+
+import re
+import shlex
+from typing import List, Tuple
+
+import cmd2
+
+BLANK_LINE = '\n'
+
+def _comment_replacer(match):
+    s = match.group(0)
+    if s.startswith('/'):
+        # treat the removed comment as an empty string
+        return ''
+    else:
```

Comment:
``else`` is unecessary since the ``if`` has a return in it

Reply:
Fixed and pushed to `ply` branch.